app/src/lib/design-system/components/RQNetworkTable/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { GenericNetworkTable } from "./GenericNetworkTable";
+import { NetworkEntry } from "./GenericNetworkTable/types";
+import { RQSessionAttributes } from "@requestly/web-sdk";
+import { secToMinutesAndSeconds } from "utils/DateTimeUtils";
+import "./RQNetworkTable.css";
+
+export interface RQNetworkLog {
+  id: string;
+  entry: NetworkEntry;
+}
+
+interface RQNetworkTableProps {
+  logs: RQNetworkLog[];
+  sessionRecordingStartTime?: RQSessionAttributes["startTime"];
+}
+
+export const RQNetworkTable: React.FC<RQNetworkTableProps> = ({ logs, sessionRecordingStartTime = 0 }) => {
+  return (
+    <div className="rq-network-table-container">
+      <GenericNetworkTable
+        logs={logs}
+        isFailed={(log: RQNetworkLog) => {
+          const { status } = log.entry.response;
+          return !status || (status >= 400 && status <= 599);
+        }}
```

Comment:
This logic can be moved in `GenericNetworkTable`. Doesn't seem to be related to Requestly

Reply:
What if others have different criteria for a failed request?